browser/brave_profile_prefs.h

```diff
@@ -1,18 +1,21 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#ifndef BRAVE_BROWSER_BRAVE_BRAVE_PROFILE_PREFS_H_
-#define BRAVE_BROWSER_BRAVE_BRAVE_PROFILE_PREFS_H_
+#ifndef BRAVE_BROWSER_BRAVE_PROFILE_PREFS_H_
+#define BRAVE_BROWSER_BRAVE_PROFILE_PREFS_H_
 
+class PrefRegistrySimple;
 namespace user_prefs {
 class PrefRegistrySyncable;
 }
 
 namespace brave {
 
+void RegisterOmniboxProfilePrefs(PrefRegistrySimple* registry);
```

Comment:
Forgot to mention - I think it would be better to move this method to related component - maybe `//brave/components/omnibox/browser`?

Reply:
:+1: